pkg/process/v5/transformers/nvd/unique_pkg.go

```diff
@@ -63,14 +63,74 @@ func findUniquePkgs(cfgs ...nvd.Configuration) uniquePkgTracker {
 	return set
 }
 
-func determinePlatformCPEAndNodes(c nvd.Configuration) (*string, []nvd.Node) {
-	var platformCPE *string
+func platformPackageCandidates(set uniquePkgTracker, c nvd.Configuration) bool {
+	nodes := c.Nodes
+	/*
+		Turn a configuration like this:
+		(AND (redis <= 6.1 (OR debian:8 debian:9 ubuntu:19 ubuntu:20))
+		Into a configuration like this:
+		(OR (AND redis <= 6.1 debian:8) (AND redis <= 6.1 debian:9) (AND redis <= 6.1 ubuntu:19) (AND redis <= 6.1 ubuntu:20))
```

Comment:
nit: your examples are for OS-related constraints, but the code is for CPEs

Reply:
These are platform CPEs, so I was using strings like `debian:9` to mean, "some CPE that says this is only vulnerable if running on debian:9." I'll look for a cleaner way to write this comment.